rb/build.desc

```diff
@@ -278,16 +279,17 @@ ruby_test(name = "unit",
     "spec/unit/selenium/webdriver/**/*_spec.rb",
     "spec/unit/selenium/*_spec.rb",
   ],
-  include = ["rb/spec/unit", "build/rb/lib"],
+  include = [
+    "rb/spec/unit",
+    "build/rb/lib"
+  ],
   deps = [
     ":chrome",
-    ":common",
```

Comment:
Have common, remote, and support gone away? or are they transitive now?

Reply:
Each browser inherits common, which inherits everything from support & remote. Seemed unnecessary to repeat it everywhere when nothing was using it.